### ISSUE TITLE
AbstractMethodError: net.minecraft.world.DimensionType.shouldLoadSpawn()Z

### ISSUE THREAD

**Issue by randombyte-developer** [SpongePowered/Sponge#1667]
**I am currently running**

- SpongeVanilla version: 1.12.2-7.0.0-BETA-383
- Java version: 1.8.0_141 OpenJDK
- Operating System: Linux (amd64) version 3.19.0-32-generic, Linux Mint

- Plugins/Mods: None

**Issue Description**

Crash report: https://gist.github.com/randombyte-developer/ddba4347ac5ef37b64c2be79803453f1

Fresh server.
It only crashes once. After another server startup it doesn't occur anymore. 
It crashes when a Player is online and only after some time, like 10 seconds, after join.
Deleting the folder `config` lets the issue appear again.

The issue appeared in earlier 1.12 versions, too. But I never reported it back then.

**Comment by stephan-gh** [comment]
Duplicate of https://github.com/SpongePowered/SpongeVanilla/issues/362
